web/index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/png" href="/token.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + Svelte</title>
+    <title>Faucet</title>
```

Comment:
Based on inputs from Jan.
```suggestion
    <title>LSK Testnet Faucet</title>
```